web/shapeworks/src/components/ShapeViewer/methods.js

```diff
@@ -0,0 +1,472 @@
+import vtkMapper from 'vtk.js/Sources/Rendering/Core/Mapper';
+import vtkRenderer from 'vtk.js/Sources/Rendering/Core/Renderer';
+import vtkSphereSource from 'vtk.js/Sources/Filters/Sources/SphereSource';
+import vtkCubeSource from 'vtk.js/Sources/Filters/Sources/CubeSource';
+import vtkImageMarchingCubes from 'vtk.js/Sources/Filters/General/ImageMarchingCubes';
+import vtkOrientationMarkerWidget from 'vtk.js/Sources/Interaction/Widgets/OrientationMarkerWidget';
+import vtkArrowSource from 'vtk.js/Sources/Filters/Sources/ArrowSource'
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+import vtkActor from 'vtk.js/Sources/Rendering/Core/Actor';
+import vtkCalculator from 'vtk.js/Sources/Filters/General/Calculator';
+import vtkCamera from 'vtk.js/Sources/Rendering/Core/Camera';
+import vtkGlyph3DMapper from 'vtk.js/Sources/Rendering/Core/Glyph3DMapper';
+import { AttributeTypes } from 'vtk.js/Sources/Common/DataModel/DataSetAttributes/Constants';
+import { ColorMode, ScalarMode } from 'vtk.js/Sources/Rendering/Core/Mapper/Constants';
+import { FieldDataTypes } from 'vtk.js/Sources/Common/DataModel/DataSet/Constants';
+
+import { getDistance } from '@/helper';
+import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
+import {
+    layers, layersShown, orientationIndicator,
+    cachedMarchingCubes, cachedParticleComparisonColors, vtkShapesByType,
+    analysisFileShown, currentAnalysisFileParticles, meanAnalysisFileParticles,
+    showDifferenceFromMeanMode, cachedParticleComparisonVectors, landmarkColorList
+} from '@/store';
+import PointDrag from './dragPoints';
```

Comment:
dragPoints isn't on this branch.

Reply:
Fixed in [14741c2](https://github.com/girder/shapeworks-cloud/pull/298/commits/14741c25be2f7f0cf31c2db266d21a754ff2182d)